pkg/datapath/loader/netlink.go

```
@@ -336,7 +336,8 @@ func attachTCProgram(link netlink.Link, prog *ebpf.Program, progName, bpffsDir s
 func removeTCFilters(ifName string, tcDir uint32) error {
 	link, err := netlink.LinkByName(ifName)
 	if err != nil {
-		return err
```

Can we be a bit more specific here? Does this return ENOENT or similar?

Discarding all errors outright always makes me a bit queasy.